java/registry/src/main/java/io/opensaber/registry/service/RegistryService.java

```diff
@@ -8,7 +8,7 @@
 
 	HealthCheckResponse health(Shard shard) throws Exception;
 
-	void deleteEntityById(Shard shard, String id) throws Exception;
+	void deleteEntityById(Shard shard, String id, String userId) throws Exception;
```

Comment:
This is exactly I wanted to avoid. Can you maintain 1 param order? it is very odd to have shard as first param, and then some data params and then finally userId.

Reply:
As you suggested, I considered userId is maintained next to shardId in all method signatures.